wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -509,6 +509,8 @@ instance FromHttpApiData WalletId where
 instance ToHttpApiData WalletId where
     toQueryParam (WalletId wid) = wid
 
+instance Hashable WalletId
```

Comment:
I am curious what the default for `Hashable` is here :/ ... I believe it picks up the `Hashable` instance for `Text` via `GeneralizedNewTypeDeriving` which is most probably fine :+1: